src/makeRobot.js

```diff
@@ -38,7 +38,62 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  return {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name}, `
+      + `chip version: ${this.version}, `
+       + `wheels: ${this.wheels}`;
```

Comment:
you could've avoid `this.` since all three are arguments passed on a function call:
```suggestion
      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
```